src/firefly/js/core/ReduxFlux.js

```diff
@@ -30,6 +30,7 @@ import DrawLayer, {DRAWING_LAYER_KEY} from '../visualize/DrawLayerCntlr.js';
 import DrawLayerFactory from '../visualize/draw/DrawLayerFactory.js';
 import DrawLayerCntlr, {makeDetachLayerActionCreator} from '../visualize/DrawLayerCntlr.js';
 import MultiViewCntlr, {IMAGE_MULTI_VIEW_KEY} from '../visualize/MultiViewCntlr.js';
+import DialogCntlr, {DIALOG_KEY} from '../core/DialogCntlr.js'
```

Comment:
And – since I am looking at this class – addListener for multiple types does not seem to be correct.